fonts.css

```diff
@@ -1,66 +1,166 @@
 }
+
 /* montserrat-500 - latin_cyrillic-ext_cyrillic */
+
 @font-face {
-    font-family: 'Montserrat';
-    font-style: normal;
-    font-weight: 500;
-    src: url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-500.eot'); /* IE9 Compat Modes */
-    src: local('Montserrat Medium'), local('Montserrat-Medium'),
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-500.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-500.woff2') format('woff2'), /* Super Modern Browsers */
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-500.woff') format('woff'), /* Modern Browsers */
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-500.ttf') format('truetype'), /* Safari, Android, iOS */
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-500.svg#Montserrat') format('svg'); /* Legacy iOS */
+  font-family: 'Montserrat';
+  font-style: normal;
+  font-weight: 500;
+  src: url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-500.eot');
+  
+  /* IE9 Compat Modes */
+  
+  src:
+          local('Montserrat Medium'), local('Montserrat-Medium'),
+          url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-500.eot?#iefix') format('embedded-opentype'),
+    
+    /* IE6-IE8 */
+  
+      url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-500.woff2') format('woff2'),
+  
+    /* Super Modern Browsers */
+  
+      url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-500.woff') format('woff'),
+    
+    /* Modern Browsers */
+  
+      url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-500.ttf') format('truetype'),
+    
+    /* Safari, Android, iOS */
+  
+      url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-500.svg#Montserrat') format('svg');
+  
+  /* Legacy iOS */
 }
+
+
 /* montserrat-600 - latin_cyrillic-ext_cyrillic */
+
 @font-face {
-    font-family: 'Montserrat';
-    font-style: normal;
-    font-weight: 600;
-    src: url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-600.eot'); /* IE9 Compat Modes */
-    src: local('Montserrat SemiBold'), local('Montserrat-SemiBold'),
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-600.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-600.woff2') format('woff2'), /* Super Modern Browsers */
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-600.woff') format('woff'), /* Modern Browsers */
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-600.ttf') format('truetype'), /* Safari, Android, iOS */
-    url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-600.svg#Montserrat') format('svg'); /* Legacy iOS */
+  font-family: 'Montserrat';
+  font-style: normal;
+  font-weight: 600;
+  src: url('./fonts/montserrat-v14-latin_cyrillic-ext_cyrillic-600.eot');
+  
+  /* IE9 Compat Modes */
+  
+  src:
+          local('Montserrat SemiBold'), local('Montserrat-SemiBold'),
```

Comment:
Codacy found an issue: [Expected newline after "," in a multi-line list (value-list-comma-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Project_AnastassiaVarabei_Evgen/pullRequest?prid=6131199)